src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+}
+
+img {
+  position: absolute;
+  top: 45px;
+  left: 26px;
+}
+
+.image-small {
+  width: 11px;
+  top: 125px;
+  left: 26px;
+}
+
+.input {
+  display: flex;
+
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  margin: 20px auto;
+  padding-left: 62px;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-small {
+  height: 42px;
+  font-size: 14px;
+  font-weight: 400px;
+  padding: 11px 0 14px 33px;
+  background-size: 11px;
+}
+
+.input::placeholder {
+
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input-small::placeholder {
+  font-size: 14px;
+}
+
+.input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```

Comment:
you already have this for `.input` class

Reply:
Ok I left box-shadow only (because of the different opacity 0.2)